rosdep/python.yaml

```diff
@@ -1843,6 +1843,19 @@ python-graphviz-pip:
 python-gridfs:
   debian: [python-gridfs]
   fedora: [python-pymongo-gridfs]
+python-grip:
+  debian:
+    pip:
+      packages: [grip]
+  fedora:
+    pip:
+      packages: [grip]
+  gentoo:
+    pip:
+      packages: [grip]
```

Comment:
It looks like this is available on gentoo as a system package: https://packages.gentoo.org/packages/app-text/grip

Reply:
Thank you! I updated.